tools/test

```diff
@@ -35,6 +35,16 @@ Options:
   --fix       Fix issues found, where possible.
   --all-files
               Run on all files, not only changed files.
+  --platform <ios|android|both|sloppy>
```

Comment:
nit: Let's keep `--all-files` next to `--diff`, because they're closely related -- in particular because they're two possible values of the same underlying option.